.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
+
+services:
+  - docker
+
+language: go
+
+go:
+  - 1.8.3
+
+before_install:
+  - sudo apt-get install libseccomp-dev -qq
+
+before_script:
+  - mkdir -p $HOME/gopath/src/github.com/aveshagarwal
+  - mv $TRAVIS_BUILD_DIR $HOME/gopath/src/github.com/aveshagarwal
```

Comment:
could you change these paths to kubernetes-incubator now?

Reply:
I can do that. In fact, I was pinging you today morning for the same reason. I think, you are working on a PR where you want to change name from rescheduler to descheduler, we can include your name removal as part of that. But for this PR, I will change and re-run the tests and see.